### ISSUE TITLE
getting err_cert_authority_invalid

### ISSUE THREAD

**Issue by latompa** [basecamp/powprox#11]
After running the bin/powprox install, 
chrome (and safari) is not trusting the cert, I get a

```
NET::ERR_CERT_AUTHORITY_INVALID

Subject: Pow development server

Issuer: Pow development root
```

I do see a the test root CA cert `Pow development root` in my Keychain and it is trusted.

My workaround is to manually import the server cert and mark it as trusted.

Is there any way around this?

**Comment by jeremy** [comment]
Looks like Keychain changes on Sierra / High Sierra. Fixed @ 65168e0bf17e56716b9a7a7148d18ebeaabbeefa

* https://openradar.appspot.com/29271915
* https://github.com/puma/puma-dev/issues/84#issuecomment-341501499

**Comment by latompa** [comment]
thank you @jeremy, that works!
